### PR TITLE
Release/1.3.0

### PR DESCRIPTION
**Pull request by nilo-ms**
## Proposed changes

Release/1.3.0

## Type of change

- [ ] Feature work
- [ ] Bug fix
- [ ] Documentation
- [ ] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

## Risk

- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [ ] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [ ] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+## [1.3.0]
+
```
Comment (kaisong1990) [inline CHANGELOG.md:2]: Is this expected to be empty?

Comment (nilo-ms) [reply CHANGELOG.md:2]: Yes, given that we're not yet in public preview we can't add anything about native auth in the changelog. We will update this file after the public preview date that is the 8th of March

**Review by kaisong1990 — APPROVED** [review]
Releasing related changes looks good. Approved

File: CHANGELOG.md (hunk 1 above)
Comment (kaisong1990) [reply CHANGELOG.md:2]: Synced offline and this is expected for private preview

**Comment by kaisong1990** [comment]
MSAL automation has been validated earlier from another merged in branch.

**Pull request merged**